help/destinations/catalog/email-marketing/salesforce-pardot.md

```diff
@@ -0,0 +1,203 @@
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+>
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate audience data to batch profile export destinations](/help/destinations/ui/activate-batch-profile-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Salesforce Pardot] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. 
+
+To correctly map your XDM fields to the [!DNL Salesforce Pardot] destination fields, follow the steps below.
+
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select custom attributes]** category and specify from the [!DNL Salesforce Pardot API](https://pi.pardot.com/api/v5/objects/prospects/do/upsertLatestByEmail) the required attribute name.
+
+    * Repeat these steps to add any additional mappings between your XDM profile schema and [!DNL Salesforce Pardot]:
+        |Source Field|Target Field| Mandatory|
+        |---|---|---|
+        |`IdentityMap: Email`|`Identity: email`| `Mandatory` |
+
+    * An example with the above mapping is shown below:
+    ![Platform UI screenshot example showing Target mappings.](../../assets/catalog/email-marketing/salesforce-pardot/mappings.png)
+
```

Comment:
Can other attributes be mapped as well or is it only possible to map identities? 
e.g. can a user map first names, last names, addresses, birthdays, etc?

Reply:
Yes, other attributes can also be updated, have linked to the API's fields page, updated the screenshot and the mappings table to showcase additional fields.